integration-tests/src/common/java/pl/allegro/tech/hermes/integrationtests/setup/HermesExtension.java

```diff
@@ -59,4 +73,46 @@ public void cutOffConnectionsBetweenBrokersAndClients() {
     public void restoreConnectionsBetweenBrokersAndClients() {
         kafka.restoreConnectionsBetweenBrokersAndClients();
     }
+
+    private void removeSubscriptions() {
+        SubscriptionService service = management.subscriptionService();
```

Comment:
I think it is beneficial in two ways:
- easier to implement
- should be more reliable (no http client involved)

The downside is that we are using implementation details (services) to clean up the storage. If they were to change test would need to be updated - but I think it is unlikely that it will happen in the foreseeable future